dlp/src/test/java/dlp/snippets/JobsTests.java

```diff
@@ -106,6 +106,18 @@ public void testCreateJobs() throws Exception {
     assertThat(output, CoreMatchers.containsString("Job created successfully."));
   }
 
+  @Test
+  public void testGetJobs() throws Exception {
+    // Create a job with a unique UUID to be gotten
+    String jobId = UUID.randomUUID().toString();
+    createJob(jobId);
```

Comment:
Great - should this be deleted at some point as these tests run many times / day?

Reply:
> Great - should this be deleted at some point as these tests run many times / day?

Thanks for bringing this up, I've updated the get and create tests to delete the created instance after the tests are done. To do that, I also made a minor change to JobsCreate.java